### PR TITLE
import data from csv and save feature added

### PR DESCRIPTION
**Pull request by codebind-luna**

### CODE REVIEW

**Comment by code-R** [comment]
- Can you add exception handling ? 
for example 
1) invalid file
2) database authentication error etc., 
- Please follow pep8 styling guideline
- Write code more organized (may be split the logic in to classes)

**Comment by code-R** [comment]
Can you use some orm solution instead of using sql queries

File: import_and_save.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        csv_data = csv.reader(file(filename))
+        lst = ['Name', 'Age', 'Address']
+        table_name = "People"
+        createsqltable = """CREATE TABLE IF NOT EXISTS """ + table_name + " (" + " VARCHAR(250),".join(lst) + " VARCHAR(250))"
+        cur.execute(createsqltable)
+        for row in csv_data:
+            cur.execute('INSERT INTO People(Name,Age,Address)''VALUES(%s, %s, %s)',row)
+        mydb_conn.commit()
+        cur.close()
+    except:
```
Comment (code-R) [inline import_and_save.py:23]: this is a bad practice, don't catch every exception here.. you should catch the exceptions you expect.

**Comment by codebind-luna** [comment]
Sure. I will do that and raise another pull request once I am done.

File: import_and_save.py (hunk 1 above)
Comment (codebind-luna) [reply import_and_save.py:23]: you mean to say some specific exceptions?

File: import_and_save.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        print('Sorry Something went wrong')
+
+def save_into_sqlite(filename):
+    try:
+        import sqlite3
+        con = sqlite3.connect("my_test.db")
+        cur = con.cursor()
+        cur.execute("CREATE TABLE IF NOT EXISTS t(Name, Age, Address);") # use your column names here
```
Comment (code-R) [inline import_and_save.py:31]: duplicate name, age, address.. If I want to change the field age as dob, I have to change every other place.

File: import_and_save.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        cur.executemany("INSERT INTO t (Name, Age, Address) VALUES (?, ?, ?);", to_db)
+        con.commit()
+        con.close()
+    except:
+        print('Sorry Something went wrong')
+
+def save_into_postgres(filename, username, password):
+    try:
+        module = import_module('psycopg2')
+        from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
```
Comment (code-R) [inline import_and_save.py:45]: imports should be defined at top of the file, please use pep8 style guideline.

File: import_and_save.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        db_name = 'cli_test'
```
Comment (code-R) [inline import_and_save.py:46]: db_name is duplicated.

File: import_and_save.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
+import click
+import csv
+from importlib import import_module
+
+def save_into_mysql(filename, username, password):
```
Comment (code-R) [inline import_and_save.py:5]: Too much logic in single function, break in to smaller functions and re-use them as necessary.

**Review by code-R — CHANGES_REQUESTED** [review]

File: import_and_save.py (hunk 1 above)
Comment (code-R) [reply import_and_save.py:23]: yes, you should catch only exceptions you expect. This makes harder to debug, you don't know if the issue is related to opening file/database operation.

**Comment by ansumanbebarta** [comment]
Git commit message could have been better. I try to follow https://chris.beams.io/posts/git-commit/

File: import_and_save.py (hunk 3 above)
Comment (codebind-luna) [reply import_and_save.py:45]: I didn't want every time all the modules to import thats why did inside the method

**Comment by code-R** [comment]
this is old

**Pull request closed without merging**